src/App.tsx

```diff
@@ -15,37 +15,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
-  return (
-    <div className="section content">
-      <div className="buttons">
-        <button type="button" className="button is-info is-light">
-          Sort alphabetically
-        </button>
-
-        <button type="button" className="button is-success is-light">
-          Sort by length
-        </button>
-
-        <button type="button" className="button is-warning is-light">
-          Reverse
-        </button>
-
-        <button type="button" className="button is-danger is-light">
-          Reset
-        </button>
-      </div>
+export enum SortType {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  DEFAULT = '',
+}
+
+interface AppState {
+  isReversed: boolean;
+  sortType: SortType;
+}
+
+export class App extends React.Component<{}, AppState> {
+  state: AppState = {
+    isReversed: false,
+    sortType: SortType.DEFAULT,
+  };
+
+  getSortedGoods = (goods: string[]): string[] => {
+    const { sortType, isReversed } = this.state;
+    const sortedGoods = [...goods];
+
+    if (sortType === SortType.ALPHABET) {
+      sortedGoods.sort();
```

Comment:
The sorting logic for alphabetical order is correct, but it might be beneficial to use `localeCompare` for more accurate sorting, especially if you expect to handle strings with special characters or different cases.